### PR TITLE
Add new tests for datepicker.

### PR DESCRIPTION
**Pull request by knight-bubble**
https://community.openproject.org/work_packages/19892

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hirtie-maxim-feature-19892-datepicker-tests.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://hirtie-maxim-feature-19892-datepicker-tests.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/tests/integration/specs/work-packages/details-pane/details-pane-helper.js [hunk 1]
```diff
@@ -38,7 +39,92 @@ function showAll() {
...
+      before(function() {
+        editor.$('.inplace-editing--trigger-link').click();
+      });
+
+      context('dropdown', function() {
+        it('should be rendered', function() {
+          expect(
+            editor
+            .$('.select2-container').isDisplayed()
+            .isDisplayed()
```
Comment (floriank) [inline frontend/tests/integration/specs/work-packages/details-pane/details-pane-helper.js]: Why is  a second call on the return value of `isDisplayed` necessary here?

Comment (knight-bubble) [inline frontend/tests/integration/specs/work-packages/details-pane/details-pane-helper.js]: I can take a look, at that. But it's not my code.

Comment (0xF013) [inline frontend/tests/integration/specs/work-packages/details-pane/details-pane-helper.js]: I removed that call twice already but it still pops up

Comment (floriank) [inline frontend/tests/integration/specs/work-packages/details-pane/details-pane-helper.js]: I removed it as well and the test still runs fine - i think it's a copy pasta error.

Log story short - remove it, after that this is good to merge :)

**Pull request merged**